run_experiments.py

```diff
@@ -334,6 +334,12 @@ def check_project(project: dict, project_dir: str, config: dict,
                            "-Xclang record-coverage=%s " % coverage_dir)
             conf_sources = [config["CodeChecker"], project, run_config]
             args.write(collect_args("clang_sa_args", conf_sources))
+
+        tidy_args_file, tidy_args_filename = tempfile.mkstemp(text=True)
```

Comment:
Since we do not always use `tidy_args_file`, maybe we should not create a temporary when additional tidy args are not set.

Reply:
Oh, I thought we always add `-Xclang -analyzer-stats`. But looking at the code again looks like that is not the case. Yeah, I guess we should do the same with `saargs`.